src/config/constants.js

```diff
@@ -112,3 +112,8 @@ export const PARAMETER_TYPES_DEFAULT = {
   [PARAMETER_TYPES.INTEGER_INPUT]: 0,
   [PARAMETER_TYPES.STRING_INPUT]: '',
 };
+
+export const DEFAULT_TAG_STYLE = {
+  backgroundColor: '#555555',
+  color: 'white',
```

Comment:
Is it possible to use `generateTextColorFromBackground` and get the background from `theme`?